src/assets/css/main.css

```diff
@@ -1529,6 +1529,10 @@ label {
   font-weight: 900;
   margin-bottom: 45px;
 }
+.contactus-brief > ul, li {
+  list-style-type: disc !important;
+  margin: 0 3rem;
```

Comment:
This broke the layout of the nav-bar links in the header. @ozzievee see attached image.
![image](https://user-images.githubusercontent.com/13269277/51380933-72a22d80-1ad8-11e9-8219-0c906be62a83.png)